parser/MyParser.java

```diff
@@ -221,23 +247,26 @@ public void extractTasks(String title, String url, String content, int outIndex)
       for (Task task : tasks) {
         numTasksAfterFiltering++;
         keptTasks.add(task.taskString);
-        bw.write(task.taskString);
-        bw.write("\n");
+        try {
+            PreparedStatement st = db.prepareStatement("INSERT INTO task (compute_index, task, search_result_content_id) VALUES (?, ?, ?)");
+            st.setInt(1, computeIndex);
+            st.setString(2, task.taskString);
```

Comment:
Each task also has a `lemmatizedTaskString` variable that is the same as the task, but with verbs lemmatized. Do we want that instead, or to store it in addition to `taskString` in the database?